hedera-node/docs/design/services/service-configuration.md

```diff
@@ -0,0 +1,47 @@
+# Service Configuration
+
+The bulleted list below provides a comprehensive overview of the various configuration settings used in the Hedera Services. These settings are defined in various Config classes,
+each managing a specific aspect of the system. For instance,
+`AccountsConfig.java` controls account-related properties, while
+`ContractsConfig.java` manages smart contract settings.
+This document serves as a valuable resource for understanding how
+Hedera Services can be customized and configured to meet specific requirements.
+For more detailed information on other types of configuration, please refer to the
+hedera-node/docs/design/services/service-configuration.md
+[Service Configuration](hedera-node/docs/services-configuration.md) file.
+
+- **AccountsConfig.java**: Provides configuration settings for various account-related properties, including admin roles and treasury accounts.
+- **ApiPermissionConfig.java**: Manages permissions for different API functionalities, specifying which accounts are allowed to execute certain operations.
+- **AutoCreationConfig.java**: Configures settings related to the automatic creation of accounts.
+- **AutoRenew2Config.java**: Contains settings for automatic renewal processes, specifying scan and renewal parameters.
+- **AutoRenewConfig.java**: Defines configurations for the auto-renewal feature, including target types for auto-renewal.
```

Comment:
I'm curious if these 2 AutoRenew Configs could be merged into one? 🤔

Reply:
Going to just leave as two lines.